src/main/java/redis/clients/jedis/JedisPool.java

```diff
@@ -27,7 +27,7 @@ public JedisPool(String host, int port) {
 
   public JedisPool(final String host) {
     URI uri = URI.create(host);
```

Comment:
Seems like we can call this(new GenericObjectPoolConfig(), URI.create(host)); instead of extracting partial URL values to remove duplication.
Could you please check? :)